pkg/dnsman2/controller/source/common/dnsspecinput.go

```diff
@@ -189,6 +176,20 @@ func augmentFromCommonAnnotations(annotations map[string]string, input DNSSpecIn
 		input.Ignore = a
 	}
 
+	if v := annotations[dns.AnnotationResolveTargetsToAddresses]; v != "" {
+		input.ResolveTargetsToAddresses = ptr.To(v == "true")
+	}
+
+	if a := annotations[dns.AnnotationCNameLookupInterval]; a != "" {
+		interval, err := strconv.ParseInt(a, 10, 64)
+		if err != nil {
+			return nil, fmt.Errorf("invalid CNameLookupInterval: %s", err)
```

Comment:
nit: We could wrap the error
```suggestion
			return nil, fmt.Errorf("invalid CNameLookupInterval: %w", err)
```